drivers/clock_control/Kconfig.stm32h7

```diff
@@ -5,6 +5,16 @@
 
 if SOC_SERIES_STM32H7X
 
+# Oscillator clocks configuration options
+
+config CLOCK_STM32_HSI_DIVISOR
+	int "HSI Divisor"
+	default 1
+	range 1 8
```

Comment:
I am not expert in Kconfig, but I wonder whether there is a way list allowed value (and avoid user to set 3 for example)

Reply:
> 
> 
> I am not expert in Kconfig, but I wonder whether there is a way list allowed value (and avoid user to set 3 for example)

Me neither 🙃 , I'll investigate on this and make the change if this is an available option.

Reply:
@lochej  You can at least add following help on the new line right after `range` keyword:
```
help Accepted values are: 1, 2, 4, 8
```